vms/avm/block/executor/manager.go

```diff
@@ -174,12 +174,7 @@ func (m *manager) VerifyTx(tx *txs.Tx) error {
 		State: stateDiff,
 		Tx:    tx,
 	}
-	err = tx.Unsigned.Visit(executor)
-	if err != nil {
-		return err
-	}
-
-	return m.VerifyUniqueInputs(m.preferred, executor.Inputs)
```

Comment:
Note: `BuildBlock` still verifies unique inputs (https://github.com/ava-labs/avalanchego/blob/verify-against-accept/vms/avm/block/builder/builder.go#L141)

This PR only removes this check from the mempool verifier.